dm/templates/iam_member/iam_member.py.schema

```diff
@@ -35,40 +39,52 @@ required:
   - roles
```

Comment:
folredID, OrgID, projectID shall not be used at the same time. Please specify this so there will be no blind overwrite.

Reply:
Done.

Reply:
This is not a problem, you can use all three if you want to. This template supports it.